presto-hive/src/main/java/io/prestosql/plugin/hive/HiveConnectorFactory.java

```diff
@@ -27,7 +27,7 @@
 import io.prestosql.plugin.hive.metastore.HiveMetastoreModule;
 import io.prestosql.plugin.hive.s3.HiveS3Module;
 import io.prestosql.plugin.hive.security.HiveSecurityModule;
-import io.prestosql.plugin.hive.security.PartitionsAwareAccessControl;
+import io.prestosql.plugin.hive.security.SystemTableAwareAccessControl;
```

Comment:
We should actually keep this rename in the same commit, since the behavior is changing along with the name. Doing it before or after would be misleading, as the behavior would not match the name.